Apps/Src/SendTritium.c

```diff
@@ -638,29 +645,25 @@ void Task_SendTritium(void *p_arg){
     state = FSM[NEUTRAL_DRIVE];
     prevState = FSM[NEUTRAL_DRIVE];
 
-    #ifndef __TEST_SENDTRITIUM
-    CANbus_Init(MOTORCAN);
+    CANbus_Init(MOTORCAN, NULL, 0);
```

Comment:
The initialization shouldn't be done in the task

Reply:
Took it out